<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArgInfoGUI</class>
 <widget class="QWidget" name="ArgInfoGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="argLayout">
    <item>
     <widget class="QLabel" name="argLabel">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Label</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="argCheck">
      <property name="styleSheet">
       <string notr="true">QCheckBox::indicator::unchecked {background: rgb(79,79,79);}
QCheckBox::indicator::checked {background: rgb(255, 157, 38);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="argEdit">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { background: rgb(50, 50, 50); border: 1px solid rgb(79, 79, 79) }</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="argCombo">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="argUnits">
      <property name="text">
       <string>Unit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
